core/services/chainlink/config.go

```diff
@@ -42,6 +42,8 @@ type Config struct {
 	Solana solcfg.TOMLConfigs `toml:",omitempty"`
 
 	Starknet stkcfg.TOMLConfigs `toml:",omitempty"`
+
+	Aptos map[string]any `toml:",omitempty"`
```

Comment:
```
type RawConfig map[string]any
```
then use
```
Aptos []*RawConfig `toml:",omitempty"`
```
since there can be multiple configs defined

Reply:
thanks for the note, added this